mk/config.mk

```diff
@@ -531,13 +531,14 @@ CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= $(CFG_ENABLE_EMBEDDED_TESTS)
 # Compiles bget_main_test() to be called from a test TA
 CFG_TA_BGET_TEST ?= $(CFG_ENABLE_EMBEDDED_TESTS)
 
-# CFG_DT_DRIVER_EMBEDDED_TEST when enabled embedb DT driver probing tests.
-# This also requires embeddeding a DTB with expected content.
-# Defautl disable CFG_DRIVERS_CLK_EARLY_PROBE to probe clocks as other drivers.
+# CFG_DT_DRIVER_EMBEDDED_TEST when enabled embedded DT driver probing tests.
```

Comment:
s/embedded/embeds/

I'm nitpicking but the change in this makefile are not related the GPIO framework tests. Could  you move them to a specific commit?
@tprrt, be sure that I really appreciate you do fix these typos. I hope you don't feel bad because I makes such comments requesting you yet another update of your series.

Reply:
Done